src/agent/rustjail/src/process.rs

```diff
@@ -41,7 +62,10 @@ pub struct Process {
     pub exit_watchers: Vec<Sender<i32>>,
     pub oci: OCIProcess,
     pub logger: Logger,
-    pub epoller: Option<Epoller>,
+    pub term_exit_notifier: Arc<Notify>,
+
+    readers: HashMap<StreamType, Reader>,
```

Comment:
What do these `readers` and `writers` are used for?

Reply:
I convert fds of stdin/stdout/... to readers and writers, store them in struct process